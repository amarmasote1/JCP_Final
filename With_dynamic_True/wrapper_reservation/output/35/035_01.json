{
    "warnings": [],
    "message": "Successful",
    "nodeType": null,
    "body": null,
    "key": "JCP|2626",
    "status": 221
}