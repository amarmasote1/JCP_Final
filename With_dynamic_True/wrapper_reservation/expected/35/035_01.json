{
    "warnings": [],
    "message": "Successful",
    "body": null,
    "key": "JCP|2626",
    "status": 221
}